anyway/models.py

```diff
@@ -921,6 +923,34 @@ def get_id(self):
         return self.id
 
 
+class LocationVerificationHistory(Base):
+    __tablename__ = "location_verification_history"
+    id = Column(BigInteger(), primary_key=True)
+    user_id = Column(BigInteger(), ForeignKey("users.id"), nullable=False)
+    news_flash_id = Column(BigInteger(), ForeignKey("news_flash.id"), nullable=False)
+    location_verification_before_change = Column(Integer(), nullable=False)
+    location_before_change = Column(Text(), nullable=False)
+    location_verification_after_change = Column(Integer(), nullable=False)
+    location_after_change = Column(Text(), nullable=False)
```

Comment:
Does it make sense to use varchar instead of text? I understand text field consumes a lot of space, and makes the retrieval slower.

Reply:
i can change this, but i think that varchar requrie to set a limit for the text size and i am not sure what is the max size for location

Reply:
OK. lets go for text now.
Each location field as both Hebrew string name and integer code. I was hoping to store the codes, but I see that the `resolution_dict` holds the hebrew text fields. Later on we will need to write a conversion from the texts to the codes.